diff --git a/node_modules/israeli-bank-scrapers/lib/scrapers/base-beinleumi-group.js b/node_modules/israeli-bank-scrapers/lib/scrapers/base-beinleumi-group.js
index 4e821fa..40517f0 100644
--- a/node_modules/israeli-bank-scrapers/lib/scrapers/base-beinleumi-group.js
+++ b/node_modules/israeli-bank-scrapers/lib/scrapers/base-beinleumi-group.js
@@ -60,6 +60,7 @@ function createLoginFields(credentials) {
 function getAmountData(amountStr) {
   let amountStrCopy = amountStr.replace(_constants.SHEKEL_CURRENCY_SYMBOL, '');
   amountStrCopy = amountStrCopy.replaceAll(',', '');
+  amountStrCopy = amountStrCopy.replace(/\s+/g, '');
   return parseFloat(amountStrCopy);
 }
 function getTxnAmount(txn) {
diff --git a/node_modules/israeli-bank-scrapers/lib/scrapers/max.js b/node_modules/israeli-bank-scrapers/lib/scrapers/max.js
index 901eba8..e49a683 100644
--- a/node_modules/israeli-bank-scrapers/lib/scrapers/max.js
+++ b/node_modules/israeli-bank-scrapers/lib/scrapers/max.js
@@ -131,6 +131,9 @@ function getTransactionType(planName, planTypeId) {
     case MaxPlanName.Credit:
     case MaxPlanName.CreditOutsideTheLimit:
       return _transactions2.TransactionTypes.Installments;
+    case "הלוואה ברגע למולטי":
+    case "הלוואה בהטבה":
+      return _transactions2.TransactionTypes.Installments;
     default:
       switch (planTypeId) {
         case 2:
